UNRELEASED_CHANGELOG.md

```diff
@@ -79,6 +79,7 @@
 
 ## stream-chat-android-compose
 ### 🐞 Fixed
+- Added missing `emptyContent` and `loadingContent` parameters to `MessageList` inner components.
```

Comment:
A quick note here: for changelog updates, always think about the user's perspective. For example, for the fix here, "Fixed emptyContent and loadingContent parameters not being respected in MessageList" might be a more suitable message.

Reply:
Ok, sure will keep the tone in mind next time